database.json

```diff
@@ -53,6 +53,12 @@
     "repo": "api",
     "desc": "Promise based HTTP client for the Deno. ༼ つ ◕_◕ ༽つ"
   },
+  "appwrite": {
+    "type": "github",
+    "owner": "appwrite",
+    "repo": "sdk-for-deno",
+    "desc": "Official Deno SDK for the Appwrite open-source backend server 🚀"
+  },
```

Comment:
not ok. I'll close the PR, just try with a fresh branch.

Reply:
Sorry, I'm still getting a hang with pull requests
Sure, will do :)